src/components/AvatarWithImagePicker.js

```diff
@@ -12,10 +12,9 @@ import themeColors from '../styles/themes/default';
 import AttachmentPicker from './AttachmentPicker';
 import AvatarCropModal from './AvatarCropModal/AvatarCropModal';
 import OfflineWithFeedback from './OfflineWithFeedback';
-import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import useLocalize from '../hooks/useLocalize';
 import variables from '../styles/variables';
 import CONST from '../CONST';
-import SpinningIndicatorAnimation from '../styles/animation/SpinningIndicatorAnimation';
```

Comment:
@keisyrzk Remove `SpinningIndicatorAnimation` file, it isn't used in the codebase.

Reply:
ok done